### PR TITLE
password reset: error message user if identifier not registered

### PR DESCRIPTION
**Pull request by ksuess**
Fix https://github.com/plone/volto/issues/2469

### CODE REVIEW

**Comment by ksuess** [comment]
What's the policy for translations in reducers? I think it's a no go policy. 
So, where can I place a translation?

**Comment by tiberiuichim** [comment]
@sneridagh @giuliaghisini

File: src/reducers/users/users.js [hunk 1]
```diff
@@ -174,6 +173,17 @@ export default function users(state = initialState, action = {}) {
...
         },
       };
+    case `${RESET_PASSWORD}_FAIL`:
+      return {
+        ...state,
+        [getRequestKey(action.type)]: {
+          loading: false,
+          loaded: false,
+          error: {
+            message: 'No user registration found for this email address.',
```
Comment (giuliaghisini) [inline src/reducers/users/users.js:183]: ok, but could you change with an intl message to be translatable?

**Comment by ksuess** [comment]
I thougt about this, if the error message should be inline or maybe better be a toast.

this is what this PR does:
image

What's the policy for error and info messages? @timo, I've seen you issued a related here: https://github.com/plone/volto/issues/1921

**Comment by sneridagh** [comment]
@ksuess ummm the login uses a toast, at least currently, because of the outcome should be present when the route changes, to the next step/state (logged in), is that the case? ... in order to make them consistent, for now I would make it a toast as well. Although I see that it might also be sensible to have it inline if there's no next state...

@plone/volto-team insights?

**Comment by ksuess** [comment]
I missed the feedback to the user, that the email is not registered, which he requests to reset the password for. Now I needed to tweak some more as replacing labels by field placeholders, theming and replacing the inline field info with a consistent use of toast messages.

So I ended with two new packages. One for a general form with placeholders and red bottom border for required fields and no labels anymore: https://github.com/rohberg/volto-tweaks-rohberg
And another package that provides the forms for Login, Password Reset and also the contact page with the above sleek form.: https://github.com/rohberg/volto-rohberg-login-tweak

So I am out here. If the above would be of general interest, just ping me.


Form

**Comment by tiberiuichim** [comment]
@ksuess Can I pursuade you to add your packages to the volto addons list here? https://github.com/collective/awesome-volto

**Pull request closed without merging**